tox.ini

```diff
@@ -18,3 +18,35 @@ commands =
     python test.py
     python setup.py sdist
     python setup.py bdist_wheel
+
+[testenv:coverage]
+commands =
+    coverage run --source src test.py
```

Comment:
Which python env will this be using ?

Reply:
Good point. Which one should it use? 3.6 maybe since 2.7 will be deprecated on 2020 and 3.6 is newer?

Reply:
3.6

Reply:
Ok, I think this is ready one more time. :)